--- node_modules/react-native/React/Modules/RCTStatusBarManager.m.orig	2019-08-27 18:02:35.000000000 -0700
+++ node_modules/react-native/React/Modules/RCTStatusBarManager.m	2019-08-27 18:03:21.000000000 -0700
@@ -138,4 +138,17 @@
 
 #endif //TARGET_OS_TV
 
+#if TARGET_OS_TV
+
+RCT_EXPORT_METHOD(getHeight:(RCTResponseSenderBlock)callback)
+{
+    callback(@[@{
+                   @"height": @(0),
+                   }]);
+}
+
+
+
+#endif
+
 @end
